adcc/ExcitedStates.py

```diff
@@ -131,7 +142,8 @@ def format(self, vector):
 
 
 class ExcitedStates:
-    def __init__(self, data, method=None, property_method=None):
+    def __init__(self, data, method=None, property_method=None,
+                 excitation_energy_corrections=None):
```

Comment:
`excitation_energy_corrections={}`

Reply:
I'd rather not do this, because it makes the default argument mutable from the outside. But I guess in this specific case, it does not do us any harm 👍

Reply:
Done.